successgilli/politico
# Ft create route user signup 163745583

#### What does this PR do?
It creates a route for the user to signup
#### Description of Task to be completed?
Have the endpoint working

- POST api/v1/auth/signup
#### How should this be manually tested?
after cloning the repo, cd into it, run 'npm i' and type npm run test.
The endpoint can also be tested using Postman or your browser
when the signup form is posted to the above route, it should return a token for the user.
if email of user exist, it should return a message 'email already exist'
#### Any background context you want to provide?
Not yet
#### What are the relevant pivotal tracker stories?
#163745583
#### Screenshots (if appropriate)
none
#### Questions:
none
